aspnetcore/signalr/supported-platforms.md

```diff
@@ -0,0 +1,37 @@
+---
+title: ASP.NET Core SignalR supported platforms
+author: rachelappel
+description: Supported platforms for ASP.NET Core SignalR 
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/20/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/supported-platforms
+---
+
+# ASP.NET Core SignalR supported platforms
+
+## Server system requirements
+
+SignalR for ASP.NET Core supports any server platform ASP.NET Core supports.
```

Comment:
...any server platform **supported by** ASP.NET Core ~~supports~~. **When using IIS, the WebSockets transport requires IIS 8.0 or higher, on Windows Server 2012 or higher. Other transports are supported on all platforms.**